### PR TITLE
git-bisect: update Spanish translation

### PR DESCRIPTION
**Pull request by kant**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.es/common/git-bisect.md [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
-> Git salta de un lado a otro del gráfico de commits para hasta alcanzar progresivamente el commit defectuoso.
+> Utiliza la búsqueda binaria para encontrar la confirmación que introdujo un error.
+> Git salta de un lado a otro del gráfico de confirmaciones hasta alcanzar progresivamente la confirmación defectuosa.
 > Más información: <https://git-scm.com/docs/git-bisect>.
 
-- Comienza un sesión de bisecado en un rango de commits delimitada por un commit erróneo conocido y por uno sano conocido (normalmente más antiguo):
+- Comienza un sesión de bisecado en un rango de confirmaciones delimitada por una confirmación erróneo conocido y por uno sano conocido (normalmente más antiguo):
 
-`git bisect start {{commit_erroneo}} {{commit_bueno}}`
+`git bisect start {{confirmacion_erroneo}} {{confirmacion_bueno}}`
```
Comment (tricantivu) [inline pages.es/common/git-bisect.md:9]:
```suggestion
`git bisect start {{confirmación_errónea}} {{confirmación_buena}}`
```

File: pages.es/common/git-bisect.md [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
 
-- Para cada commit que `git bisect` selecciona, marcarlo como "malo" o "bueno" después de probarlo para el problema:
+- Por cada confirmación que `git bisect` selecciona, lo marca como "malo" o "bueno" después de probarlo:
 
 `git bisect {{bueno|malo}}`
```
Comment (tricantivu) [inline pages.es/common/git-bisect.md:13]: As far as I know, `git bisect` does not recognize the arguments `bueno` and `malo`. If you want, create another pull to fix this @kant.

```suggestion
`git bisect {{good|bad}}`
```

**Review by tricantivu — COMMENTED** [review]
This changes are done in #12046.

**Review by gutjuri — APPROVED** [review]

**Review by kbdharun — COMMENTED** [review]
Closing this PR, as the page in main is already up to date with these changes.

**Pull request closed without merging**